client/post-editor/controller.js

```diff
@@ -115,6 +115,7 @@ module.exports = {
 
 		function startEditing( siteId ) {
 			context.store.dispatch( setEditorPostId( postID ) );
```

Comment:
Also just a small remark, I'm wondering why the lingering edits change the behavior of the "New" links. They seem unrelated to me. Maybe a bug on these links instead no?